### PR TITLE
Absence record model updated with created students field

### PR DESCRIPTION
**Pull request by dting**
- Add school with no record to seed.js
- Small refactor for absence record controller to
  reduce repeated code.

### CODE REVIEW

File: client/app/main/pdf-upload/pdf-upload.controller.js [hunk 1]
```diff
@@ -51,21 +51,24 @@ function PDFUploadCtrl($scope, PDF, AbsenceRecord, Auth, School, toastr) {
     }, $scope.parsedRecord, function(res) {
       resetState();
       var schoolName = res.record.school.name;
-      if (res.students.length) {
+      if (res.outreaches.length) {
+        toastr.info(
+          res.outreaches.length + ' outreaches triggered.',
+          schoolName,
+          {timeOut: 5000}
+        );
```
Comment (pdotsani) [inline client/app/main/pdf-upload/pdf-upload.controller.js]: I noticed that some of our toastrs have a dismiss feature and others don't. Is there a common format we want to follow in terms of timeout and dismissible, or is it just a case by case basis?

File: server/api/absence-record/absence-record.controller.js [hunk 2]
```diff
@@ -73,38 +45,66 @@ exports.validateCreate = function(req, res, next) {
...
       newEntries[index].student = student._id;
     });
-    return newAbsenceRecord({
+    var combinedEntries = [].concat.apply(newEntries, existingEntries);
+    // Outreaches for entries are update with student and added to outreaches.
+    _.forEach(combinedEntries, function(entry) {
+      _.forEach(entry.outreaches, function(outreach) {
+        outreach.student = entry.student;
+        outreaches.push(outreach);
+      });
```
Comment (pdotsani) [inline server/api/absence-record/absence-record.controller.js]: This is fine. But I've always been curious if there's a more elegant, and efficient way to go about handling cases in which you do need to use nested forEach statements:
http://stackoverflow.com/questions/21532912/nested-lodash-foreach

**Comment by pdotsani** [comment]
lgtm. Just a few comments.

File: client/app/main/pdf-upload/pdf-upload.controller.js (hunk 1 above)
Comment (dting) [inline client/app/main/pdf-upload/pdf-upload.controller.js]: Case by case.

File: server/api/absence-record/absence-record.controller.js (hunk 2 above)
Comment (dting) [inline server/api/absence-record/absence-record.controller.js]: The link you have is still using nested foreach. We aren't creating objects so the optimization they suggest isn't relevant.

**Pull request merged**